torch/csrc/jit/codegen/cuda/kernel_cache.h

```diff
@@ -107,8 +104,7 @@ class CudaKernelCache {
   CudaKernelCache() = default;
 
   at::optional<CudaKernel*> getKernelPtr(
-      const at::ArrayRef<c10::IValue> inputs,
-      const std::vector<int64_t>& broadcasted_shape);
+      const at::ArrayRef<c10::IValue> inputs);
```

Comment:
const ref

Reply:
kernel cache will get rewritten or removed in following prs.